backend/x86_binary_emitter.ml

```diff
@@ -1398,20 +1403,17 @@ let emit_FLDCW b = function
   | (Mem _ | Mem64_RIP _) as rm -> emit_mod_rm_reg b no_rex [ 0xD9 ] rm 5
   | _ -> assert false
 
-let emit_FXCH b = function
-  | Regf (ST i) -> buf_opcodes b [ 0xD9; 0xC8 + i ]
+let emit_FXCH _b = function
```

Comment:
Can these instructions be removed too? If so, can you please make a separate PR for the changes in registerf (removal of TOS and ST and the corresponding instructions)?

Reply:
PR #1555